cpp-ch/local-engine/local_engine_jni.cpp

```diff
@@ -699,6 +699,60 @@ JNIEXPORT jlong Java_io_glutenproject_vectorized_CHShuffleSplitterJniWrapper_nat
     LOCAL_ENGINE_JNI_METHOD_END(env, -1)
 }
 
+JNIEXPORT jlong Java_io_glutenproject_vectorized_CHShuffleSplitterJniWrapper_nativeMakeForRSS(
+    JNIEnv * env,
+    jobject,
+    jstring short_name,
+    jint num_partitions,
+    jbyteArray expr_list,
+    jbyteArray out_expr_list,
+    jint shuffle_id,
+    jlong map_id,
+    jint split_size,
+    jstring codec,
+    jlong spill_threshold,
+    jobject pusher)
+{
+    LOCAL_ENGINE_JNI_METHOD_START
+    std::string hash_exprs;
+    std::string out_exprs;
+    if (expr_list != nullptr)
+    {
+        int len = env->GetArrayLength(expr_list);
+        auto * str = reinterpret_cast<jbyte *>(new char[len]);
+        memset(str, 0, len);
```

Comment:
No need memset, since we write data later with true value, right?

Reply:
right